packages/react-devtools/app.html

```diff
@@ -146,11 +146,27 @@
     </div>
     <script>
       const {clipboard} = require("electron");
-      const port = process.env.PORT || 8097;
+      const secure = process.env.SECURE && process.env.SECURE === 'true';
```

Comment:
nit
```js
const secure = process.env.SECURE === 'true';
```

Reply:
This line has been removed as a result of your previous comments.